frontend_app_file/src/index.js

```diff
@@ -8,19 +8,37 @@ import PopupCreateFile from './container/PopupCreateFile.jsx'
 
 require('./css/index.styl')
 
+// FIXME - GB - 2021-05-26 - Both if in the appInterface const are workarounds for the asynchrony problem of the renderAppFeature and unmountApp functions.
+// The code of the functions relies on the _reactRootContainer property to know whether the element is a React element or not and
+// should be rendered or unmounted (see https://github.com/facebook/react/blob/7841d0695ae4bde9848cf8953baf34d312d0cced/packages/react-dom/src/client/ReactDOMLegacy.js).
+// The workaround ensures that a new render is only executed if the unmount of the previous elements has been executed and vice versa.
+// More Information: https://github.com/tracim/tracim/issues/4679
+
 const appInterface = {
   name: 'file',
   isRendered: false,
-  renderAppFeature: (data) => {
+  renderAppFeature: data => {
+    const element = document.getElementById(data.config.domContainer)
+    if (element._reactRootContainer) {
+      console.log('Component is already mounted, waiting for unmount!!', appInterface)
+      setTimeout(() => { appInterface.renderAppFeature(data) }, 50)
+      return
+    }
     return ReactDOM.render(
       <Router history={data.config.history}>
         <File data={data} />
       </Router>
-      , document.getElementById(data.config.domContainer)
+      , element
     )
   },
   unmountApp: domId => {
-    return ReactDOM.unmountComponentAtNode(document.getElementById(domId)) // returns bool
+    const element = document.getElementById(domId)
+    if (domId === 'appFeatureContainer' && !element._reactRootContainer) {
+      console.log('Component is not mounted, waiting for mount!!', appInterface)
```

Comment:
```suggestion
      console.log('WORKAROUND (#4679): Component is not mounted, waiting for mount!!', appInterface)
```

or:

```suggestion
      console.log('HACK (#4679): Component is not mounted, waiting for mount!!', appInterface)
```

Maybe? (everywhere if you decide to apply)

Reply:
Fixed